### PR TITLE
Fix issue of initiallyOpen in @polar/plugin-icon-menu not working

### PR DESCRIPTION
**Pull request by dopenguin**
## Summary

See the commit for more information.

## Instructions for local reproduction and review

- `npm run meldemichel:dev`
- As configured, the `layerChooser` should be open on start

### CODE REVIEW

**Review by warm-coolguy — CHANGES_REQUESTED** [review]
This is missing a changelog entry.

🏓 @dopenguin

File: packages/plugins/IconMenu/src/store/index.ts [hunk 1]
```diff
@@ -45,16 +45,11 @@ export const makeStoreModule = () => {
...
-        if (!rootGetters.hasSmallWidth && !rootGetters.hasSmallHeight) {
-          dispatch('openMenuById', initiallyOpen)
-        }
       },
-      openMenuById({ commit, getters, dispatch }, openId) {
-        const index = getters.menus.findIndex(({ id }) => id === openId)
+      openMenuById({ commit, dispatch, getters }) {
+        const index = getters.menus.findIndex(
+          ({ id }) => id === getters.initiallyOpen
+        )
```
[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/plugins/IconMenu/src/store/index.ts]: Done 2d6a87e6d0f7110dfed2fffc288be2d7d93f78f5

**Comment by dopenguin** [comment]
> This is missing a changelog entry.
> 
> 🏓 @dopenguin

Added 3c26939e54aaad34dfabc53786ba80284ea17f84

🏓 @warm-coolguy

**Review by warm-coolguy — APPROVED** [review]
🚀 @dopenguin

**Pull request merged**